### DISCUSSION TITLE
Segmentation Error

### DISCUSSION THREAD

**Discussion by nowlabs** [qdm12/gluetun#2168] [Q&A] [answered]
Hello, I just updated gluetun this morning and now I am getting this segmentation error below

Any assistance is appreciated

2024-03-19T14:53:56Z INFO [firewall] enabling...
2024-03-19T14:53:56Z INFO [firewall] enabled successfully
2024-03-19T14:53:57Z INFO [storage] merging by most recent 17820 hardcoded servers and 17820 servers read from /gluetun/servers.json
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x760835]

goroutine 7 [running]:
github.com/qdm12/gluetun/internal/configuration/settings.(*ServerSelection).validate(0xc0000a7f30, {0xc00002c035, 0x7}, {0x1098740?, 0xc0024e9680?})
	github.com/qdm12/gluetun/internal/configuration/settings/serverselection.go:149 +0xa55
github.com/qdm12/gluetun/internal/configuration/settings.(*Provider).validate(0xc0000a7f28, {0xc00002a021?, 0x9?}, {0x1098740, 0xc0024e9680})
	github.com/qdm12/gluetun/internal/configuration/settings/provider.go:47 +0x2fc
github.com/qdm12/gluetun/internal/configuration/settings.(*VPN).Validate(0xc0000a7f18, {0x1098740, 0xc0024e9680}, 0xe0?)
	github.com/qdm12/gluetun/internal/configuration/settings/vpn.go:30 +0x165
github.com/qdm12/gluetun/internal/configuration/settings.(*Settings).Validate.func1()
	github.com/qdm12/gluetun/internal/configuration/settings/settings.go:52 +0x2b
github.com/qdm12/gluetun/internal/configuration/settings.(*Settings).Validate(0xc0000a7c00, {0x1098740?, 0xc0024e9680}, 0x0)
	github.com/qdm12/gluetun/internal/configuration/settings/settings.go:57 +0x9cd
main._main({0x109d840, 0xc000490000}, {{0x1096954, 0x6}, {0x1096cd8, 0x7}, {0x1098470, 0x18}}, {0xc00008e050, 0x1, ...}, ...)
	./main.go:240 +0xf9d
main.main.func1()
	./main.go:92 +0x12c
created by main.main in goroutine 1
	./main.go:91 +0x5e5

**Comment by frepke** [comment, marked as answer]
There are already multiple issues about this error, for now you can change from `:latest to :v3` as a temporary solution.

**Comment by frepke** [comment]
The problem is solved now, you can pull the `latest` and run the container.

**Comment by qdm12** [comment]
Thanks @frepke (also extra thanks for whatsapp-ing me to let me know this blew up 😄)

It's now solved indeed, just repull the image.

**Reply by frepke** [reply]
I thought you hadn't caught on yet, and I believe you would want to resolve it quickly if you knew 😊
